monai/bundle/scripts.py

```diff
@@ -192,6 +193,13 @@ def _download_from_ngc(
     extractall(filepath=filepath, output_dir=extract_path, has_base=True)
 
 
+def _download_from_huggingface_hub(repo: str, download_path: str, filename: str, version: str) -> None:
+    if len(repo.split("/")) != 2:
+        raise ValueError("if source is `huggingface_hub`, repo should be in the form `repo_owner/repo_name`")
+    extract_path = os.path.join(download_path, filename)
+    huggingface_hub.snapshot_download(repo_id=repo, revision=version, local_dir=extract_path, local_dir_use_symlinks="auto")
```

Comment:
```suggestion
    huggingface_hub.snapshot_download(repo_id=repo_id, revision=version, local_dir=extract_path)
```

`local_dir_use_symlinks="auto"` is not wrong but it's already the default value for that so you can forget it :)

Reply:
I wanted to explicit here about the use of symlinks since it might be unexpected behavior for the user (given that it's different from the other methods for downloading bundles in MONAI), but explaining this behavior in the docstring would probably be a better move.

Reply:
Makes sense yes!